### PR TITLE
add configuration for .well-known/webfinger

### PR DESCRIPTION
**Pull request by daita**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Let's wait for 
https://github.com/nextcloud/server/pull/11396 first

File: admin_manual/installation/nginx.rst [hunk 1]
```diff
@@ -94,6 +94,9 @@ webroot of your nginx installation. In this example it is
       #rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json
       # last;
 
+      # Uncomment this line to enable webfinger (used by ActivityPub)
+      rewrite ^/.well-known/webfinger /public.php?service=webfinger last;
```
Comment (tcitworld) [inline admin_manual/installation/nginx.rst]: Shouldn't it be commented then ?

**Comment by h3ndrik** [comment]
hey, thanks for the hint. i believe this should be merged now as the advertised "social" app needs webfinger.

**Comment by daita** [comment]
@nextcloud/documentation ping !

**Comment by juliushaertl** [comment]
Already done by @violoncelloCH https://github.com/nextcloud/documentation/pull/1024/files

**Pull request closed without merging**